pkg/tinyproxy

```diff
@@ -1,9 +1,10 @@
 [mirrors]
-https://github.com/tinyproxy/tinyproxy/releases/download/1.8.4/tinyproxy-1.8.4.tar.xz
+https://github.com/tinyproxy/tinyproxy/releases/download/1.11.0/tinyproxy-1.11.0.tar.xz
 
 [vars]
-filesize=192300
-sha512=23398a2c8a6d926b371086ba96032d0fd8dd06d114edf24950b868f53bb6b4235cd0f5d6b9a0131879fcc16bbe6b71142a6855de593a937ef7b0c323b50e0aec
+filesize=177664
+sha512=c0ff2a90da2352e6021efc87d1293b85f2a7588b2a1ab13bd96d906382eda8188b22b01f9acb2d200fd570bf3ec9484fa370cae6834d1f055c0af1631d22ae9d
+pkgver=2
 desc="lightweight HTTP proxy"
 
 [deps]
```

Comment:
you should be able to remove line 23-30, there are no longer any hacks needed for clean build